admin/modules/user/users.php

```diff
@@ -992,7 +992,7 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".$mybb->settings['useravatar'];
+		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
That's how everything works... Check smilies, post icons, etc. You need to point to a path and `$theme` is not available in ACP.